tools/circle_plus_gen/schema/circle_traininfo_generated.py

```diff
@@ -601,9 +602,38 @@ def LossReductionType(self):
             return self._tab.Get(flatbuffers.number_types.Int8Flags, o + self._tab.Pos)
         return 0
 
+    # ModelTraining
```

Comment:
`circle_traininfo.fbs` is updated here - https://github.com/Samsung/ONE/pull/13000.
So, Let's update flatc generated files.